core/src/test/scala/unit/kafka/tools/StorageToolTest.scala

```diff
@@ -559,6 +548,27 @@ Found problem:
     assertEquals(expectedOutput.trim, output.trim)
   }
 
+  @Test
+  def testTestingFeatureDependencies(): Unit = {
+    val stream = new ByteArrayOutputStream()
+    val namespace = StorageTool.parseArguments(Array("feature-dependencies", "--feature", "test.feature.version=2"))
+
+    StorageTool.runFeatureDependenciesCommand(namespace, new PrintStream(stream), testingFeatures)
```

Comment:
testingFeature is never enabled in the tool. Do we need to test this functionality?

Reply:
No other features have dependencies so I wanted to test that the code to find and print dependencies works.